DALK.PL_ANALYZER/Content/css/vendor/dataTables.bootstrap4.css

```diff
@@ -0,0 +1,206 @@
+table.dataTable {
```

Comment:
same here. This is probably fetched from nuget or external library, so this should never-ever landed in repsitory